source/content/guides/decoupled/no-starter-kit/04-any-framework.md

```diff
@@ -69,4 +69,128 @@ You can use frontend frameworks other than Gatsby and Next.js. Note that while o
     - Environment variables
     - Deployment path
 
+1. Click **Build Details** to view the build log.
+
+## Bootstrap Site Creation Example
```

Comment:
Is there a reason Bootstrap was chosen for this example? Nothing wrong with it, but from my perspective this could be seen as an "older" tech and if we want to portray a sense of cutting edge, I would not put an example for Bootstrap in the docs.

I would rather see this guide fitted to 'No framework' or 'no build' – how to use plain html/css/js for your Front-End Site. In that case the user can choose to use bootstrap.

Reply:
@CobyPear I'll update the section to create a no framework html/css/js site. I'd like to add an SSR template example. Do you have any recommendations? Recommendations for other popular SSG examples would also be appreciated.

Reply:
Vite has a CLI to scaffold starters for a lot of libraries with SSR capabilities -> https://github.com/vitejs/vite/tree/main/packages/create-vite#create-vite. The `react-ts` one might be a good start for SSR.

Maybe the `lit` or `lit-ts` could be a good static example? Almost any library can be built into a static site. Are you specifically looking for another SSG like docusaurus?